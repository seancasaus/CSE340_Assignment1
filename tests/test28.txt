FHMIZ (((f)*).(x))|(I) , gr (r).((T)|((b).(l))) , rP4XO ((R).((3)|(w)))|(b) , GJyK1 ((6).((G)|(W)))|(((7)|(Q))*) , LSGa5 h #
" I6GrT6G6GrTrblbQIbrThbh" 
