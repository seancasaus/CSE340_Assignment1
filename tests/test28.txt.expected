FHMIZ , "I"
GJyK1 , "6G"
gr , "rT"
GJyK1 , "6G"
GJyK1 , "6G"
gr , "rT"
gr , "rbl"
rP4XO , "b"
GJyK1 , "Q"
FHMIZ , "I"
rP4XO , "b"
gr , "rT"
LSGa5 , "h"
rP4XO , "b"
LSGa5 , "h"
